pom.xml

```diff
@@ -60,7 +60,7 @@
         <hadoop.version>2.7.3</hadoop.version>
         <quartz.version>2.3.2</quartz.version>
         <jackson.version>2.10.5</jackson.version>
-        <mybatis-plus.version>3.2.0</mybatis-plus.version>
+        <mybatis-plus.version>3.4.0</mybatis-plus.version>
```

Comment:
It's better not upgrade `mybatis-plus` in this PR, If you hope to do this, it's necessary to create a new issue and discuss with community first.

Reply:
sorry about that, i choose the way of adding a typeHandler to fix issue #7732 . but due to the mybatis-plus [BUG](https://github.com/baomidou/mybatis-plus/issues/2550), it failed in the maven test. so i update a oldest version that works.

Reply:
We can discuss in #7732 .